backend/src/main/kotlin/com/tngtech/apicenter/backend/config/AclConfiguration.kt

```diff
@@ -0,0 +1,147 @@
+package com.tngtech.apicenter.backend.config
+
+import org.springframework.boot.jdbc.DataSourceBuilder
+import org.springframework.cache.CacheManager
+import org.springframework.cache.support.NoOpCacheManager
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.jdbc.core.JdbcTemplate
+import org.springframework.jdbc.datasource.DataSourceTransactionManager
+import org.springframework.security.acls.jdbc.BasicLookupStrategy
+import org.springframework.security.acls.jdbc.JdbcMutableAclService
+import org.springframework.security.acls.AclPermissionEvaluator
+import org.springframework.security.acls.domain.*
+import org.springframework.security.acls.model.PermissionGrantingStrategy
+import org.springframework.security.core.authority.SimpleGrantedAuthority
+import javax.sql.DataSource
+import org.springframework.security.core.context.SecurityContextHolder
+import org.springframework.security.core.authority.AuthorityUtils
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken
+import org.springframework.transaction.support.AbstractPlatformTransactionManager
+
+@Configuration
+class AclConfiguration {
+
+    @Bean
+    fun transactionManager(
+            dataSource: DataSource
+    ): AbstractPlatformTransactionManager {
+        return DataSourceTransactionManager(dataSource)
+    }
+
+    @Bean
+    fun dataSource(): DataSource {
+        val dataSource = DataSourceBuilder
```

Comment:
All of this is okay for local tests, but we need something more robust for actual production environments. For example, I can't imagine we will use H2 there. Also, can we move the SQL scripts to an external file and maybe use flyway or something?